Dockerfile

```diff
@@ -1,7 +1,16 @@
-FROM squidfunk/mkdocs-material:6.2.4
```

Comment:
Was anything more on that image that we don't have on this? Just to make sure we would not introduce a different behavior.
Also, how we can make sure pip will install a specific version of mkdocs-material, and not latest? We already had issues in the past we breaking changes like that

Reply:
> Was anything more on that image that we don't have on this? Just to make sure we would not introduce a different behavior.

I'm not a python ecosystem expert, but as far as I can tell the only difference is it's installing mkdocs-material through pip not through git which (for whatever reason) is what blocks the theme overrides from working.

> how we can make sure pip will install a specific version of mkdocs-material, and not latest?

Same as the packages below, I should be able to do `mkdocs-material=6.2.4` or something, I'll double check that works with pip